### PR TITLE
test(task): run test from intercepted response

### PR DESCRIPTION
**Pull request by blegesse-git**
Addresses #2887 

`Can clone a task and edit it` test fails with `Expected to find content: 'import "csv"' within the element: <div.flux-editor--monaco> but never did.` 

Extending the time to wait for `.contains('import "csv")` to resolve might be a solution for this flakiness. 

The solution above didn't work. 

Implementing Timmy's suggestion -> intercept the api call and wait for response to populate.

### CODE REVIEW

**Comment by TCL735** [comment]
This is okay to try.

If this does not work, then another thing to try is to intercept the api calls and wait on them. Then, ideally, the text would be populated.

**Review by TCL735 — APPROVED** [review]

**Comment by blegesse-git** [comment]
> This is okay to try.
> 
> If this does not work, then another thing to try is to intercept the api calls and wait on them. Then, ideally, the text would be populated.

thanks for the suggestion, timmy. I'll definitely look into that

File: cypress/e2e/shared/tasks.test.ts [hunk 1]
```diff
@@ -274,29 +274,32 @@ from(bucket: "defbuck")
...
       cy.getByTestID('task-form-name').should('have.value', '🦄ask (clone 1)')
       cy.getByTestID('task-form-name')
+        .focus()
         .clear()
         .type('Copy task test')
       cy.getByTestID('task-form-name').should('have.value', 'Copy task test')
 
       cy.getByTestID('task-form-schedule-input').should('have.value', '24h')
       cy.getByTestID('task-form-schedule-input')
+        .focus()
```
Comment (blegesse-git) [inline cypress/e2e/shared/tasks.test.ts:295]: same here

File: cypress/e2e/shared/tasks.test.ts [hunk 2]
```diff
@@ -274,29 +274,32 @@ from(bucket: "defbuck")
...
         .clear()
         .type('12h')
       cy.getByTestID('task-form-schedule-input').should('have.value', '12h')
 
       cy.getByTestID('task-form-offset-input').should('have.value', '20m')
       cy.getByTestID('task-form-offset-input')
+        .focus()
```
Comment (blegesse-git) [inline cypress/e2e/shared/tasks.test.ts:302]: ditto

**Comment by ui-ci-bot-influx** [comment]
Metric | Score | Level
------------ | ------------- | ------------- 
performance | 53 | :red_circle:
accessibility | 97 | :green_circle:
seo | 90 | :green_circle:
pwa | null | :hankey:
best-practices | 100 | :green_circle:


[Compare your scores with master branch](https://influxdb.aws.influxdata.io/orgs/844910ece80be8bc/dashboards/073c4b95f1089000?lower=now%28%29%20-%202d&vars%5Blighthouse_score_branch%5D=4007)

**Review by hoorayimhelping — APPROVED** [review]

**Pull request merged**